src/recordlinker/linking/link.py

```diff
@@ -33,21 +33,35 @@ def fhir_record_to_pii_record(fhir_record: dict) -> schemas.PIIRecord:
         "birthDate": fhir_record.get("birthDate"),
         "sex": fhir_record.get("gender"),
         "address": fhir_record.get("address", []),
-        "phone": fhir_record.get("telecom", []),
         "mrn": None,
+        "ssn": None,
+        "race": None,
+        "gender": None,
+        "telecom": fhir_record.get("telecom", []),
     }
     for identifier in fhir_record.get("identifier", []):
         for coding in identifier.get("type", {}).get("coding", []):
             if coding.get("code") == "MR":
                 val["mrn"] = identifier.get("value")
+            elif coding.get("code") == "SS":
```

Comment:
I know this doesn't all fall on this PR, but I noticed that we don't have any unit tests for `fhir_record_to_pii_record`, would you mind adding some to the test_link.py module?

Reply:
Yeah good callout. I thought of that too while doing this because of the code coverage percentage. I'll add some tests in